src/commonMain/kotlin/org/intellij/markdown/parser/constraints/CommonMarkdownConstraints.kt

```diff
@@ -263,7 +263,7 @@ open class CommonMarkdownConstraints protected constructor(private val indents:
     }
 
     override fun toString(): String {
-        return "MdConstraints: " + String(types) + "(" + indent + ")"
+        return "MdConstraints: " + types.concatToString() + "(" + indent + ")"
```

Comment:
The `String(CharArray)` constructor was remove in 1.5, `concatToString` is the equivalent replacement.